### ISSUE TITLE
No email sent when SECURITY_TEAM_EMAIL set

### ISSUE THREAD

**Issue by DataSquid** [Netflix/security_monkey#174]
Running 0.3.7, using SES.  Have email sending to individual users (both notification and password reset).  But when I populate the SECURITY_TEAM_EMAIL field nothing is sent.

**Comment by monkeysecurity** [comment]
Anything in the logs?

Did you restart the scheduler after adding the SECURITY_TEAM_EMAIL?

**Comment by monkeysecurity** [comment]
Are changes occurring in your environment?  It will only send an email when it sees an item's configuration change.

**Comment by DataSquid** [comment]
Yes, I have two other user accounts setup receiving change emails (one with the same address I've used for TEAM).  Restarted scheduler a few times now.

**Comment by DataSquid** [comment]
Does this value get used to send daily reports, or per-change emails?

**Comment by monkeysecurity** [comment]
Should be both daily reports and per-change emails.

**Comment by cbarrac** [comment]
I'm seeing a strange problem that may be the issue. I added the following line to common/utils/utils.py just above `send_email(app.config...`
```
app.logger.debug("Attempting to email from {} to {} with subject {}".format(recipient, email, subject))
```
Looking at the debug output, it's chopping an email address in to one letter chunks, and trying to send to them.
E.g. SECURITY_TEAM_EMAIL=monkey@domain
This would attempt to send emails to:
m
o
n
k
e
y
etc (one email per letter of recipient address).
Not had a chance to debug further

**Comment by cbarrac** [comment]
Debugging further, it's reading the SECURITY_TEAM_EMAIL as an array, but users in the database are fine.
```
app.logger.debug("Recipients: {}".format(recipients))
```
```
Recipients: ['m', 'o', 'n', 'k', 'e', 'y',....,u'user@domain']
```

**Comment by cbarrac** [comment]
Fix for me: line 51 of auditor.py - change `self.email.extend(self.team_emails)` to `self.email.append(self.team_emails)`
Haven't tested alerter.py (line 64), but assume the same will work there.

**Comment by cbarrac** [comment]
In my case, error was tracked down to `SECURITY_TEAM_EMAIL` being defined as a string `'email@domain.tld'` instead of an array `['email@domain.tld']`
security_monkey_auto_install.sh was setting this value incorrectly (see pull request #193)

Auditor / Alerter expanded to parse both (see pull request #194)

**Comment by monkeysecurity** [comment]
Just merged #194. I believe this fixes the issue. Closing issue.

Thanks to @cbarrac for identifying the problem.  I couldn't figure it out.
